atomicapp/cli/main.py

```diff
@@ -365,6 +375,21 @@ def create_parser(self):
             help='The name of a container image containing an Atomic App.')
         gena_subparser.set_defaults(func=cli_genanswers)
 
+        # === "init" SUBPARSER ===
+        init_subparser = toplevel_subparsers.add_parser(
+            "init", parents=[globals_parser])
```

Comment:
why do we need global_parsers? for ex, we won't be using --provider-config for init